_integration-schemas/adroll/v1/ad_groups.md

```diff
@@ -0,0 +1,118 @@
+---
+tap: "adroll"
+version: "1"
+key: ""
+
+name: "ad_groups"
+doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-adgroup-get"
+singer-schema: "https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/schemas/ad_groups.json"
+description: |
+  The `{{ table.name }}` table contains info about the adgroups contained within the campaigns of your {{ integration.display_name }} account.
+
+replication-method: "Full Table"
+
+api-method:
```

Comment:
This actually isn't the endpoint the tap is using to get adgroup data. Take a look at [`streams.py` for this table](https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/streams.py#L160):

```python
class AdGroups(Stream):
    stream_id = 'ad_groups'
    stream_name = 'ad_groups'
    endpoint = 'advertisable/get_adgroups'
    key_properties = ["eid"]
    # It seems like this endpoint now has pagination and filtering capabilities.
    replication_method = "FULL_TABLE"
    replication_keys = []
```